src/client/utils/url.js

```diff
@@ -1,279 +1,278 @@
-                                                              Const.DOM_SANDBOX_STORED_ATTR_POSTFIX) ||
-                      NativeMethods.getAttribute.call(iframe, 'src') || iframe.src;
-
-    var parsedProxyDocumentLocation = documentLocation && UrlUtil.isSupportedProtocol(documentLocation) &&
-                                      SharedUrlUtil.parseProxyUrl(documentLocation);
-    var parsedProxySrcLocation      = srcLocation && UrlUtil.isSupportedProtocol(srcLocation) &&
-                                      SharedUrlUtil.parseProxyUrl(srcLocation);
-
-    return {
-        documentLocation: parsedProxyDocumentLocation ? parsedProxyDocumentLocation.originUrl : documentLocation,
-        srcLocation:      parsedProxySrcLocation ? parsedProxySrcLocation.originUrl : srcLocation
-    };
-};
-
-function getLocation () {
-    try {
-        // NOTE: fallback to the owner page's URL if we are in the iFrame without src
-        if (window.frameElement && UrlUtil.isIframeWithoutSrc(window.frameElement))
-            return Settings.get().REFERER;
-    }
-        /*eslint-disable no-empty */
-    catch (e) {
-        // NOTE: Cross-domain iframe
-    }
-    /*eslint-enable no-empty */
-
-    return window.location.toString();
-}
-
-UrlUtil.OriginLocation = {
-    get: function () {
-        var location = getLocation();
-
-        return SharedUrlUtil.parseProxyUrl(location).originUrl;
-    },
-
-    withHash: function (hash) {
-        var location = this.get();
-
-        // NOTE: remove previous hash if we have one
-        location = location.replace(/(#.*)$/, '');
-
-        return location + hash;
-    },
-
-    getCookiePathPrefix: function () {
-        var parsedLocation = SharedUrlUtil.parseProxyUrl(getLocation());
-
-        return parsedLocation.partAfterHost.replace(parsedLocation.originResourceInfo.partAfterHost, '');
-    },
-
-    getParsed: function () {
-        var resolver     = getResolver(document);
-        var origin       = this.get();
-        var parsedOrigin = SharedUrlUtil.parseUrl(origin);
-
-        // NOTE: IE "browser" adds default port for the https protocol while resolving
-        resolver.href = this.get();
-
-        // NOTE: IE ignores first '/' symbol in the pathname
-        var pathname = resolver.pathname.indexOf('/') === 0 ? resolver.pathname : '/' + resolver.pathname;
-
-        //TODO describe default ports logic
-        return {
-            protocol: resolver.protocol,
-            // NOTE: remove default port
-            port:     parsedOrigin.port ? resolver.port : '',
-            hostname: resolver.hostname,
-            // NOTE: remove default port form the host
-            host:     parsedOrigin.port ? resolver.host : resolver.host.replace(/:\d+$/, ''),
-            pathname: pathname,
-            hash:     resolver.hash,
-            search:   resolver.search
-        };
-    }
-};
-
-export default UrlUtil;
+import NativeMethods from '../sandboxes/native-methods';

```

Comment:
Please, setup line endings in your editor to avoid this kind of diffs. We have the `.editorconfig` file which forces line endings, you just need to install editorconfig plugin for your WebStorm